package.json

```diff
@@ -14,19 +15,10 @@
     "ios",
     "android"
   ],
-  "author": {
-    "name": "Ladislav Martincik",
-    "email": "ladislav.martincik@gmail.com",
-    "url": "https://github.com/martincik"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/slowpath/react-native-hockeyapp"
-  },
-  "bugs": {
-    "url": "https://github.com/slowpath/react-native-hockeyapp/issues"
+    "url": "https://github.com/peartherapeutics/react-native-hockeyapp"
```

Comment:
you *can* if you don't use the official forking mechanism ;)
but yeah I'd personally rather do it this way anyway, otherwise we won't be able to make PRs